docs/topic/billing/things.md

```diff
@@ -0,0 +1,610 @@
+# What exactly do cloud providers charge us for?
+
+There are a million ways to pay cloud providers money, and trying to
+understand it all is [several](https://www.oreilly.com/library/view/cloud-finops/9781492054610/)
+[books](https://www.porchlightbooks.com/product/cloud-cost-a-complete-guide---2020-edition--gerardus-blokdyk?variationCode=9780655917908)
+[worth](https://www.abebooks.com/servlet/BookDetailsPL?bi=31001470420&dest=usa) [of](https://www.thriftbooks.com/w/cloud-data-centers-and-cost-modeling-a-complete-guide-to-planning-designing-and-building-a-cloud-data-center_caesar-wu_rajkumar-buyya/13992190/item/26408356/?srsltid=AfmBOootnd77xklpoo2MTy8n0np1b5oamDo5KgOg9dCD-0bKody2zEF14oU#idiq=26408356&edition=14835620)
+[material](https://bookshop.org/p/books/reduce-cloud-computing-cost-101-ideas-to-save-millions-in-public-cloud-spending-abhinav-mittal/10266848).
```

Comment:
```suggestion
There are a million ways to pay cloud providers money, and trying to
understand it all is several books worth of material.[^1][^2][^3][^4][^5]

[^1]: https://www.oreilly.com/library/view/cloud-finops/9781492054610/
[^2]: https://www.porchlightbooks.com/product/cloud-cost-a-complete-guide---2020-edition--gerardus-blokdyk?variationCode=9780655917908
[^3]: https://www.abebooks.com/servlet/BookDetailsPL?bi=31001470420&dest=usa
[^4]: https://www.thriftbooks.com/w/cloud-data-centers-and-cost-modeling-a-complete-guide-to-planning-designing-and-building-a-cloud-data-center_caesar-wu_rajkumar-buyya/13992190/item/26408356/?srsltid=AfmBOootnd77xklpoo2MTy8n0np1b5oamDo5KgOg9dCD-0bKody2zEF14oU#idiq=26408356&edition=14835620
[^5]: https://bookshop.org/p/books/reduce-cloud-computing-cost-101-ideas-to-save-millions-in-public-cloud-spending-abhinav-mittal/10266848
```

Let's make these links [footnotes](https://mystmd.org/guide/typography#footnotes) so that the source markdown is more readable.

Reply:
Great idea! I've turned these into reference style links and moved the URLs to the bottom.